### ISSUE TITLE
node_modules/@apollo/client/masking/types.d.ts:2:111 - error TS2691: An import path cannot end with a '.ts' extension.

### ISSUE THREAD

**Issue by MasterAzazel** [kamilkisiela/apollo-angular#2326]
**Describe the bug**

The apollo angular package V5.0.2 was changed recently and is not building anymore.

**To Reproduce**

-An app that built until recently
-Remove node_modules and do an npm install
-Try to build my app: node_modules/@apollo/client/masking/types.d.ts:2:111 - error TS2691: An import path cannot end with a '.ts' extension:
![image](https://github.com/user-attachments/assets/a1906e72-1772-4c80-aa45-e3808374c6b8)

**Environment:**

```
- @angular/cli@15.2.8
- @angular/core@15.2.10
- @apollo/client@3.12.0
- apollo-angular@5.0.2
- graphql@16.9.0
- typescript@4.9.5
```

**Comment by PowerKiKi** [comment]
The root cause is not in this package. See instead https://github.com/apollographql/apollo-client/issues/12170

**Comment by jerelmiller** [comment]
@MasterAzazel we just released v3.12.1 which should address this issue. Sorry for the error!
